packages/angular/projects/ui-angular/src/lib/primitives-poc/icon/icon.component/type.ts

```diff
@@ -0,0 +1,4 @@
+export interface viewstate {
+  width: number;
+  height: number;
+}
```

Comment:
Last NIT: let's capitalize `viewState` to make it obvious it's a type.

Reply:
```suggestion
export interface Viewstate {
  width: number;
  height: number;
}
```

(and other place it's used